src/NuGet.Server/Infrastructure/JsonNetPackagesSerializer.cs

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace NuGet.Server.Infrastructure
+{
+    public class JsonNetPackagesSerializer
```

Comment:
This could probably done more simply using a converter - Here is an example http://stackoverflow.com/questions/23475806/allow-empty-string-value-for-emailaddress-validation-attribute/23476319#23476319